src/arith/rewrite_simplify.cc

```diff
@@ -1026,7 +1026,9 @@ PrimExpr RewriteSimplifier::Impl::VisitExpr_(const MinNode* op) {
       int64_t c1val = c1.Eval()->value;
       int64_t c2val = c2.Eval()->value;
       if (c2val % c1val == 0) {
-        if (c2val / c1val >= 0) {
+        if (c2val == 0) {
+          return c1val > 0 ? (min(x, 0) * c1val).Eval() : (max(x, 0) * c1val).Eval();
+        } else if (c2val / c1val > 0) {
```

Comment:
Thanks @xqdan this rule seems also should applies to other cases when c1val is smaller than 0, 

e.g. (  `min(x * (-2),  -4)`), please also fix that case

Reply:
We should also fix the case for max

Reply:
We can change original line from `c2val / c1val >= 0` to `c1val > 0`.

Reply:
Let us also fix the (rare) special case, `(c1val == 0)`

Reply:
ok, fixed